### ISSUE TITLE
Why is my best IoU only 66.4%? Is there something wrong with the hyperparameter settings in my config file?

### ISSUE THREAD

**Issue by Misswcnm** [dvlab-research/SphereFormer#77]
Why is my best IoU only 664? Is there something wrong with the hyperparameter settings in my config file? I’m training on 4 A30s in a cluster

<img width="981" height="693" alt="Image" src="https://github.com/user-attachments/assets/987ae8bb-9ae2-4bef-a1e5-a6efc60b5be4" />

DATA:
  data_name: semantic_kitti
  data_root: /share/home/u2415283021/SemanticKitti/dataset  # 数据路径
  label_mapping: util/semantic-kitti.yaml
  classes: 19
  fea_dim: 6
  voxel_size: [0.05, 0.05, 0.05]
  voxel_max: 120000 

TRAIN:
  # architecture
  arch: unet_spherical_transformer
  input_c: 4
  m: 32
  block_reps: 2
  block_residual: True
  layers: [32, 64, 128, 256, 256]
  quant_size_scale: 24
  patch_size: 1 
  window_size: 6
  use_xyz: True
  sync_bn: True
  rel_query: True
  rel_key: True
  rel_value: True
  drop_path_rate: 0.3
  max_batch_points: 1000000
  class_weight: [3.1557, 8.7029, 7.8281, 6.1354, 6.3161, 7.9937, 8.9704, 10.1922, 1.6155, 4.2187, 1.9385, 5.5455, 2.0198, 2.6261, 1.3212, 5.1102, 2.5492, 5.8585, 7.3929]
  xyz_norm: False
  pc_range: [[-51.2, -51.2, -4], [51.2, 51.2, 2.4]]
  window_size_sphere: [2, 2, 80]
  window_size_scale: [2.0, 1.5]
  sphere_layers: [1,2,3,4,5]
  grad_checkpoint_layers: []
  a: 0.0125
  loss_name: ce_loss
  use_tta: False
  vote_num: 4

  # training setup
  aug: True
  transformer_lr_scale: 0.1 
  scheduler_update: step 
  scheduler: Poly
  power: 0.9
  use_amp: True
  train_gpu: [0,1,2,3]           # 使用 4 张 GPU
  workers: 16                    # 可适当加大以充分利用 CPU
  batch_size: 8
  batch_size_val: 8
  base_lr: 0.006
  epochs: 50 
  start_epoch: 0
  momentum: 0.9
  weight_decay: 0.02 
  drop_rate: 0.5

  ignore_label: 255
  manual_seed: 123
  print_freq: 100
  save_freq: 1
  save_path: runs/semantic_kitti_unet32_spherical_transformer
  weight:
  resume:
  evaluate: True
  eval_freq: 10
  val: False

Distributed:
  dist_url: env://
  dist_backend: nccl
  multiprocessing_distributed: True
  world_size: 1          # ✅ 单节点，必须是 1
  rank: 0
